sections/01_abstract.md

```diff
@@ -1,3 +1,18 @@
 ## Abstract
 
-Abstract goes here.
+Deep learning, a new class of machine learning algorithms, has recently showed
+impressive results across a variety domains. Biomedicine and genomics, both
+data- and feature-rich and yet complex and ill-understood, presents an obvious
+target for applying this new approach. We examine applications of deep learning
+to a variety of biomedical data problems - in patient classification,
+fundamental research and patient treatment -  to determine if similar progress
```

Comment:
Maybe

> fundamental biological processes

instead of

> fundamental research

Also, is the categorize section exclusively about _patient_ classification? Given that a lot of the categorization happens on images or other types of samples perhaps we can be more broad (which would also remove the repetition of the word patient).